submissions/Eugene-CG/DOM/style.css

```diff
@@ -0,0 +1,137 @@
+:root {
+  --icon-size: 61px;
+}
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+.wrapper {
+  height: 200vh;
+  display: grid;
+  grid-template: auto 1fr auto / 1fr 1fr 8fr 1fr 1fr;
+  grid-template-areas:
+    ". header header header ."
+    ". main main main ."
+    ". footer footer footer .";
+  background: linear-gradient(
+    -45deg,
+    #616fbd 0%,
+    #353170 13%,
+    #4369af 25%,
+    #331fa5 38%,
+    #00369b 50%,
+    #57fff1 62%,
+    #4ceaff 75%,
+    #0098ac 87%,
+    #006b7e 100%
+  );
+  background-size: 400% 400%;
+  animation: animate 7.5s ease-in-out infinite;
+}
+.header {
+  display: flex;
+  justify-content: space-between;
+  grid-area: header;
+}
+a {
+  color: black;
+  text-decoration: none;
+}
+.menu__popup-icon {
+  display: none;
+}
+.menu__popup-icon img {
+  height: var(--icon-size);
+  width: var(--icon-size);
+}
+.menu__list {
+  display: flex;
+  justify-content: space-around;
+  list-style: none;
+}
+.menu__list > li,
+.logo {
+  margin-right: 1em;
+  line-height: 3em;
+}
+.main {
+  grid-area: main;
+}
+.main__content {
+  margin-top: 3em;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+.main__content > p {
+  margin: 2em 0;
+  border-bottom: 3px solid blue;
+  font-size: 1.2em;
+  transition: 0.2s all ease-in-out;
+}
+.footer {
+  grid-area: footer;
+}
+.menu__list > li > a,
```

Comment:
Avoid styling by HTML tags. [Reason](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)
Also, avoid using too specific CSS selectors. [Read more](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)